<launch>

  <param name="/use_sim_time" value="true" />
  <!-- all the parameters we need -->
  <rosparam file="$(find riskrrt)/params/riskrrt_params.yaml" command="load"/>
  

  <node pkg="map_server" type="map_server" name="map_server" output="screen" args="$(find riskrrt)/worlds/test/map.yaml" respawn="false">
    <param name="frame_id" value="/map" />
  </node>
 
  <!-- Start stage -->
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(find riskrrt)/worlds/test/map.world" respawn="false" output="screen"/>
  
  <node pkg="tf" 
        type="static_transform_publisher" 
        name="robot0_tf" 
        args="-3.7 -5.0 0 1.57 0 0 /map /robot_0/odom  10" />
        
  <node pkg="tf" 
        type="static_transform_publisher" 
        name="robot1_tf" 
        args="-3.5 2 0 1.57 0 0 /map /robot_1/odom  10" />
  
  <!-- Map builder -->
  <node name="og_builder_stage" pkg="riskrrt" type="og_builder_stage" output="screen" args="1 0.3"/>

  <!-- amcl -->
  <include file="$(find riskrrt)/launch/amcl_sim.launch"/>

    
  <!-- RiskRRT Node-->
  <node pkg="riskrrt" type="planner" name="planner" output="screen">
     <remap from="/odom" to="/robot_0/odom" />
  </node>
  
  <!-- Controller Node-->
  <node pkg="riskrrt" type="controller" name="controller" output="screen">
     <remap from="/cmd_vel" to="/robot_0/cmd_vel" />
  </node>
  
  <!-- rviz -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find riskrrt)/conf/riskrrt_test.rviz"/> 

</launch>

